ui/src/app/applications/components/application-summary/application-summary.tsx

```diff
@@ -381,7 +381,7 @@ export const ApplicationSummary = (props: ApplicationSummaryProps) => {
                 if (!updatedApp.spec.syncPolicy) {
                     updatedApp.spec.syncPolicy = {};
                 }
-                updatedApp.spec.syncPolicy.automated = {prune, selfHeal};
+                updatedApp.spec.syncPolicy.automated = {prune, selfHeal, enabled: true};
```

Comment:
This is how I tested it out.  Create the App with this field 'turned on', and then check the manifest to see if it's there.  Then go to the application summary, and check that it is there in the MANIFEST tab.  Then go to the SUMMARY tab and make the change, and then check the manifest again.  And check the MANIFEST tab.

Also, separately, try changing it from the MANIFEST tab and then switch to the SUMMARY tab to see if it accurately reflects that values.